pkg/consts/constants.go

```diff
@@ -53,24 +53,27 @@ const (
 	SriovSwitchDevConfPath     = SriovConfBasePath + "/sriov_config.json"
 	SriovHostSwitchDevConfPath = Host + SriovSwitchDevConfPath
 
-	DrainAnnotationState         = "sriovnetwork.openshift.io/state"
-	DrainAnnotationStateRequired = "sriovnetwork.openshift.io/state-required"
-	DrainAnnotationTime          = "sriovnetwork.openshift.io/state-time"
-
-	DrainIdle        DrainState = "Idle"
-	DrainDisabled    DrainState = "Drain_Disabled"
-	DrainRequired    DrainState = "Drain_Required"
-	RebootRequired   DrainState = "Reboot_Required"
-	DrainMcpPausing  DrainState = "Draining_MCP_Pausing"
-	DrainMcpPaused   DrainState = "Draining_MCP_Paused"
-	Draining         DrainState = "Draining"
-	DrainingComplete DrainState = "Draining_Complete"
-	RebootComplete   DrainState = "Reboot_Complete"
+	MachineConfigPoolPausedAnnotation       = "sriovnetwork.openshift.io/state"
+	MachineConfigPoolPausedAnnotationIdle   = "Idle"
+	MachineConfigPoolPausedAnnotationPaused = "Paused"
+
+	NodeDrainAnnotation             = "sriovnetwork.openshift.io/state"
+	NodeStateDrainAnnotation        = "sriovnetwork.openshift.io/desired-state"
```

Comment:
this annotation is referred in `daemon.go` only, always with write operations. So it looks like it's never read. Can we remove it?

Reply:
that is to prepare for the next step that will move the operator to look into that annotation